cmd/cli/comm_trans.go

```diff
@@ -139,6 +142,10 @@ func (c *CommTrans) GetInvokeRequestFromDesc() (*pb.InvokeRequest, error) {
 	}
 
 	var preExeReq *pb.InvokeRequest
+	if bytes.Contains(desc, []byte("tdpos")) {
```

Comment:
so I can write a description contains "tdpos" for any transaction to enforce using multisign?

Reply:
So replace tdpos with contract_name, it's more general